misc/config.py

```diff
@@ -16,4 +16,7 @@
 # where to store JSON files
 JSON_PATH = Path.cwd() / "public/json/v1/"
 
+# maximum custom rootfs size
+MAX_CUSTOM_ROOTFS_SIZE_MB = 131072
```

Comment:
131GB of rootfs? Could you please check the size of the resulting compressed images and how long it takes to build those? It's an obvious DOS attack to create some large images and thereby filling op the server storage.

@dangowrt can you think of a better way to handle this? Usually I'd expect to have a limit of 1GB to install binaries and then another data partition.

Reply:
I'd wait for a comment of @dangowrt since he's done some prior work on partition managing. I'm fine with a max of say 512MB or 1024MB but more than that seems excessive for installed binaries.

Reply:
@dangowrt Any updates?